# Chinese (Hong Kong) translation for ubuntuone-client
# Copyright (c) 2010 Rosetta Contributors and Canonical Ltd 2010
# This file is distributed under the same license as the ubuntuone-client package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2010.
#
msgid ""
msgstr ""
"Project-Id-Version: ubuntuone-client\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2012-09-10 21:26+0000\n"
"PO-Revision-Date: 2011-08-11 17:20+0000\n"
"Last-Translator: Roy Chan <roy.chan@linux.org.hk>\n"
"Language-Team: Chinese (Hong Kong) <zh_HK@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Launchpad-Export-Date: 2012-09-17 18:32+0000\n"
"X-Generator: Launchpad (build 15966)\n"

#: ../ubuntuone/status/aggregator.py:51
msgid "Magicicada"
msgstr "Magicicada"

#: ../ubuntuone/status/aggregator.py:52
msgid "New cloud folder(s) available"
msgstr "有可用的新雲端資料夾"

#: ../ubuntuone/status/aggregator.py:53
msgid "File synchronization completed."
msgstr "檔案同步已完成。"

#: ../ubuntuone/status/aggregator.py:55
#, python-format
msgid "%(percentage_completed)d%% completed."
msgstr "已完成 %(percentage_completed)d%%。"

#: ../ubuntuone/status/aggregator.py:56
msgid "File synchronization in progress"
msgstr "檔案同步正在進行中"

#: ../ubuntuone/status/aggregator.py:59
#, python-format
msgid ""
"There is no available space on the folder:\n"
"\"%s\" shared by %s"
msgstr ""
"該資料夾沒有可用空間：\n"
"「%s」由 %s 所分享"

#: ../ubuntuone/status/aggregator.py:73
#, python-format
msgid "'%(filename)s' is being uploaded to your personal cloud."
msgstr "正在上傳「%(filename)s」至您的私有雲。"

#: ../ubuntuone/status/aggregator.py:79
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file are being uploaded to your "
"personal cloud."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files are being uploaded to your "
"personal cloud."
msgstr[0] "正在上傳「%(filename)s」和其他 %(other_files)d 個檔案至您的私有雲。"

#: ../ubuntuone/status/aggregator.py:90
#, python-format
msgid "'%(filename)s' is being downloaded to your computer."
msgstr "正在下載「%(filename)s」至您的電腦。"

#: ../ubuntuone/status/aggregator.py:96
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file are being downloaded to your "
"computer."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files are being downloaded to your "
"computer."
msgstr[0] "正在下載「%(filename)s」和其他 %(other_files)d 個檔案至您的電腦。"

#: ../ubuntuone/status/aggregator.py:107
#, python-format
msgid "'%(filename)s' was uploaded to your personal cloud."
msgstr "正在上傳「%(filename)s」至您的私有雲。"

#: ../ubuntuone/status/aggregator.py:113
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file were uploaded to your personal "
"cloud."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files were uploaded to your "
"personal cloud."
msgstr[0] "「%(filename)s」和其他 %(other_files)d 個檔案已經上傳至您的私有雲。"

#: ../ubuntuone/status/aggregator.py:124
#, python-format
msgid "'%(filename)s' was downloaded to your computer."
msgstr "已下載「%(filename)s」至您的電腦。"

#: ../ubuntuone/status/aggregator.py:130
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file were downloaded to your "
"computer."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files were downloaded to your "
"computer."
msgstr[0] "「%(filename)s」和其他 %(other_files)d 個檔案已經下載至您的電腦。"

#: ../ubuntuone/status/aggregator.py:191
#, python-format
msgid "A file was just made public at %(new_public_url)s"
msgstr "一份檔案剛被設為公開，位於 %(new_public_url)s"

#: ../ubuntuone/status/aggregator.py:204
#, python-format
msgid "%(event_count)d file was just made public."
msgid_plural "%(event_count)d files were just made public."
msgstr[0] "%(event_count)d 份檔案剛設為公開。"

#: ../ubuntuone/status/aggregator.py:212
msgid "A file is no longer published"
msgstr "一份檔案不再發布"

#: ../ubuntuone/status/aggregator.py:220
#, python-format
msgid "%(event_count)d file is no longer published."
msgid_plural "%(event_count)d files are no longer published."
msgstr[0] "%(event_count)d 份檔案不再發布。"

#: ../ubuntuone/status/aggregator.py:235
#, python-format
msgid "Found %(event_count)d new cloud folder."
msgid_plural "Found %(event_count)d new cloud folders."
msgstr[0] "找到 %(event_count)d 個新雲端資料夾。"

#: ../ubuntuone/status/aggregator.py:243
#, python-format
msgid ""
"New cloud folder available: '%(folder_name)s' shared by %(other_user_name)s"
msgstr "有可用的新雲端資料夾：由 %(other_user_name)s 分享的「%(folder_name)s」"

#: ../ubuntuone/status/aggregator.py:259
#, python-format
msgid "New cloud folder available: '%(folder_name)s'"
msgstr "有可用的新雲端資料夾：「%(folder_name)s」"

#: ../ubuntuone/status/aggregator.py:281
msgid "The connection to the server was lost."
msgstr "已失去與伺服器的連線。"

#: ../ubuntuone/status/aggregator.py:287
msgid "The connection to the server was restored."
msgstr "已恢復與伺服器的連線。"

#: ../data/emblem-ubuntuone-downloading.icon.in.h:1
msgid "Downloading"
msgstr "下載中"

#: ../data/emblem-ubuntuone-unsynchronized.icon.in.h:1
msgid "Unsynchronized"
msgstr "未同步"

#: ../data/emblem-ubuntuone-uploading.icon.in.h:1
msgid "Uploading"
msgstr "上傳中"
